libs/qCC_db/src/ccCoordinateSystem.cpp

```diff
@@ -0,0 +1,262 @@
+//##########################################################################
+//#                                                                        #
+//#                              CLOUDCOMPARE                              #
+//#                                                                        #
+//#  This program is free software; you can redistribute it and/or modify  #
+//#  it under the terms of the GNU General Public License as published by  #
+//#  the Free Software Foundation; version 2 or later of the License.      #
+//#                                                                        #
+//#  This program is distributed in the hope that it will be useful,       #
+//#  but WITHOUT ANY WARRANTY; without even the implied warranty of        #
+//#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the          #
+//#  GNU General Public License for more details.                          #
+//#                                                                        #
+//#          COPYRIGHT: Chris Brown                                        #
+//#                                                                        #
+//##########################################################################
+
+#include "ccCoordinateSystem.h"
+
+//qCC_db
+#include "ccPlane.h"
+#include "ccPointCloud.h"
+
+
+ccCoordinateSystem::ccCoordinateSystem(PointCoordinateType displayScale, PointCoordinateType axisWidth, const ccGLMatrix* transMat/*= 0*/,
+	QString name/*=QString("CoordinateSystem")*/)
+	: ccGenericPrimitive(name, transMat), m_DisplayScale(displayScale), m_width(axisWidth), m_showAxisPlanes(true), m_showAxisLines(true)
```

Comment:
could you add one initialization per line? (with the comma in front of each one)

Reply:
fixed